plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TableStatisticsReader.java

```diff
@@ -210,9 +219,9 @@ else if (columnHandle.getBaseType() == VARBINARY) {
         return new TableStatistics(Estimate.of(recordCount), columnHandleBuilder.buildOrThrow());
     }
 
-    private Map<Integer, Long> readNdvs(long snapshotId, Set<Integer> columnIds)
+    private Map<Integer, Long> readNdvs(long snapshotId, Set<Integer> columnIds, boolean extendedStatisticsEnabled)
     {
-        if (!isExtendedStatisticsEnabled(session)) {
```

Comment:
is it a prep change?

Reply:
Yeah, made the testing easier. Want it in a prep commit?

Reply:
Added

Reply:
Ah, I force-pushed and lost it...